website/docs/bestpractices/analytics/trino-best-practices.md

```diff
@@ -0,0 +1,642 @@
+  - key: "kubernetes.io/arch“
+  operator: In
+  values: ["arm64"]
+  - key: "karpenter.k8s.aws/instance-category"
+  operator: In
+  values: ["r"]
+  - key: "karpenter.k8s.aws/instance-family"
+  operator: In
+  values: ["r6g", "r7g", "r8g"]
+  - key: "karpenter.k8s.aws/instance-size"
+  operator: In
+  values: ["2xlarge", "4xlarge"]
+  disruption:
+  consolidationPolicy: WhenEmptyOrUnderutilized
+  consolidateAfter: 60s
+  limits:
+  cpu: "1000"
+  memory: 1000Gi
+  weight: 100
+  ```
+</details>
+
+Example of a Karpenter Node Pool setup can be also viewed in the [DoEKS repository](https://github.com/awslabs/data-on-eks/blob/f8dda1ae530902b77ee123661265caa09d97969b/distributed-databases/trino/karpenter.tf#L100) and the [EC2 Node Class configuration](https://github.com/awslabs/data-on-eks/blob/f8dda1ae530902b77ee123661265caa09d97969b/distributed-databases/trino/karpenter.tf#L65).
+
+### Mixed Instances
+Configure mixed instance types within the same instance pool to enhance flexibility and ensure access to a range of instance sizes, from small to large, based on workload demands.
+
+
+:::tip[Key Recommendations]
+:::
+- Use Karpenter: For better scaling and simplified management of Trino clusters, prefer using Karpenter. It provides faster node provisioning, enhanced flexibility with mixed instance types, and better resource efficiency. Karpenter delivers superior scaling capabilities compared to MNG, making it the preferred choice for scaling secondary(worker) nodes in analytics applications.
+- Dedicated Nodes: Deploy one Trino pod per node to fully utilize available resources.
+- DaemonSet Resource Allocation: Reserve enough CPU and memory for essential system DaemonSets to maintain node stability.
+- Coordinator Placement: Always run the Trino coordinator on an on-demand node to guarantee reliability.
+- Worker Distribution: Use a mix of On-Demand and Spot instances for worker nodes to balance cost-effectiveness with availability.
+- Managed Node Groups Usage : MNG should be used for components of the workloads resiliency purposes and other components of the cluster like observability tools.
+</CollapsibleContent>
+
+<CollapsibleContent header={<span><h2>Trino on EKS Setup</h2></span>}>
+Helm streamlines your Trino deployment on EKS. You can deploy using either the [official Helm chart](https://github.com/trinodb/charts) or community charts for configuration management.
+
+## Setup
+
+* Install Trino using the official Helm chart or community charts
+* Create distinct Helm releases for each cluster (ETL, Interactive, BI)
+
+### Configuration Steps
+
+* Define unique `values.yaml` files per cluster
+* Configure pod resources:
+  * Set CPU/memory requests
+  * Set CPU/memory limits
+  * Specify coordinator resources
+  * Specify worker resources
+* Set pod scheduling:
+  * Configure nodeSelector
+  * Configure affinity rules
+
+### Deployment
+
+Use separate Helm commands for each workload type, with their respective values files. For example
```

Comment:
I would use the phrase "helm configurations" instead of helm commands because it's usually a better idea to use CD systems instead of helm cli.

Reply:
Agreed